assignment.md

```diff
@@ -12,46 +12,86 @@ If the version control concepts or `git` commands are still confusing to you, co
 
     ```bash
     # git command used and new list of branches
+     git checkout -b submission
+
+      main
+      * submission
     ```
 
  1. List the current `remote` name and hyperlink destination of this assignment repository (Hint: if you are not sure, add `--help` after your command to check remotes.). 
 
     ```bash
     # your git command and output here
+    git remote -v
+    origin	https://github.com/mrb4002/version_control_assignment.git
     ```
 
 
  2. Add a second remote destination named `upstream` to your assignment repository, `upstream` is a common name used for the repository that people have forked from, in our case, the `upstream` will be at `https://github.com/WCM-datascibasics/version_control_assignment`. List the new list of remotes after you've added the new remote. 
  
     ```bash
     # list the new list of remotes here
+    origin
+    upstream
     ```
 
  3. Save your modifications to this file so far and create a commit indicating you've answered the first 2 questions. Then try pushing the changes to the `upstream` destination on GitHub. What happens? Explain in your own words why does this happen? What are the benefits of having this `upstream` remote added when working collaboratively (Hint: read git command outputs!)?
 
-    > Your answer here
+    > When I try to push the changes to 'upstream' it denies access because I don't have permission to push to that remote. The benefits of having the 'upstream' remote when working collaboratively is that you can pull other people's work.
```

Comment:
Also to keep one public working copy away from testing modifications that may or may not work properly.